GetPet/GetPet.WebApi/Controllers/OrganizationsController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AutoMapper;
+using GetPet.BusinessLogic.Model;
+using GetPet.BusinessLogic.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GetPet.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class OrganizationsController : ControllerBase
+    {
+        private readonly IMapper _mapper;
+        private readonly ILogger<OrganizationsController> _logger;
+        private readonly IOrganizationRepository _organizationRepository;
+
+        public OrganizationsController(
+            ILogger<OrganizationsController> logger,
+            IMapper mapper,
+            IOrganizationRepository organizationRepository)
+        {
+            _logger = logger;
+            _mapper = mapper;
+            _organizationRepository = organizationRepository;
+        }
+
+        [HttpGet]
+        public async Task<IEnumerable<OrganizationDto>> Get([FromQuery] BaseFilter filter)
```

Comment:
u need to return Task<IActionResult> and wrap the result in OK()
implement also post
check model validation and throw a relevant error like 'Email already exists'..